blog/2021-12-14-cohort-based-modelling/index.md

```diff
@@ -0,0 +1,69 @@
+
+## Example of cohort analysis
+
+An example of settings of Apache COOL is as follows:
+
+![example of settings](./img/settings.png)
+
+Here, only patients diagnosed with disease B will be selected for the analysis. The birth action for patients is taking medicine A twice. The time unit of age is one day. The metric is to count patients with abnormal values in lab-test C. For each patient, the measured period, the range of the age, is the following 7 days after taking medicine A twice. Patients are selected into different cohorts according to their birth year.
+
+Finally, we could obtain the analysis results as follows:
+
+![result of line map](./img/result_line_map.png)
+
+![result of heat map](./img/result_heat_map.png)
+
+In the line map, each line stands for a cohort in which the patients are born in the same decade. The line map could not only illustrate the trend of patients' behavior along the time but also offer a view of the difference between different cohorts.
+
+The heat map is presented along with age and cohorts. Different colors give spontaneous expression on the evolvement of patient behavior and indicate deep insight into patient behavior between different cohorts.
+
+From the two charts, we can observe that younger patients are easier to exhibit side effects, while elder patients take longer to get accustomed to the medicine.
+
+## The applications of Apache COOL with AI
+
+### Is the criterion for cohorts effective?
+
+There are many criteria to divide the cohorts. So how to evaluate the cohorts? It's a hard problem because there are no regular distance metrics and evaluations about patients and cohorts. We could combine Apache COOL with metric learning to find an appropriate distance metric to measure the similarities between patients. Then we could measure the cohorts with the average similarities in the same cohorts and between different cohorts. In this way, the more similar the patients in the same cohort are and the more different the patients in different cohorts are, the better the criterion is. Therefore, we could perform cohort analyses as many as we want and select the most effective criterion to dig deeper.
+
+For example, we may have several assumptions to find what factors will influence mortality after admission to the ICU in 48 hours. Then we could perform cohort analyses with Apache COOL according to our assumptions and compare the effectiveness of different assumptions. Finally, we could precisely find the important factors.
+
+### Processing for missing values
+
+There are many missing values in Electronic Health Record(EHR). We have many methods to deal with missing values, such as mean imputation, hot deck imputation and regression imputation. Now we have a new method to estimate the missing values with cohorts analysis. We could replace the missing values by averaging the corresponding values of the patients in the same cohort. We should note that this method could be applied to other estimation tasks. 
```

Comment:
The first sentence should be the topic sentence of the paragraph, and it should be more informative and can tell readers what this paragraph is about.

Reply:
suggestion: example diagrams can make these parts more interesting and attractive.